doc/README.md

```diff
@@ -189,8 +192,47 @@ paths with the highest cum weight.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+*  **_cum_ value**, the value of a location plus all of its descendants, is indicated by the nodes' color.
+
+    *   Redder nodes have larger total values.
```

Comment:
This and below still uses total / self terminology.

Reply:
Done.